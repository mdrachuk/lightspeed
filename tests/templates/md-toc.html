<html lang="en">
<head><title>{{ markdown.title }}</title></head>
<body>
{{ markdown.toc_html }}
</body>
</html>