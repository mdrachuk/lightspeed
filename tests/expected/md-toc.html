<html lang="en">
<head><title>None</title></head>
<body>
<ul id="table-of-content">
<li><a href="#this-is-a-test">This is a test</a>
<ul>
<li><a href="#second-heading">Second heading</a>
<ul>
<li><a href="#third-heading">Third heading</a></li>
</ul>
</li>
</ul>
</li>
</ul>

</body>
</html>